### PR TITLE
Amna J.s week1

### PR DESCRIPTION
**Pull request by AmnaAhmad**

### CODE REVIEW

File: Week1/homework/js-exercises/exercise2.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+'use strict';
+console.log("I'm awesome")
```
Comment (marcianoviereck) [inline Week1/homework/js-exercises/exercise2.js:2]: Don' forget the `;` :)

File: Week1/homework/js-exercises/exercise5.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+'use strict';
+const z = 7.25;
+console.log(z);
+const a = 7;
+console.log(a);
+const biggistNumber = a > z ? a : z;
```
Comment (marcianoviereck) [inline Week1/homework/js-exercises/exercise5.js:6]: Ah nice, you can also use [Math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max)!

File: Week1/homework/js-exercises/exercises10.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+const firstArray = ["good", 26, true, {name: "Amna"}];
+const secondArray = ["beter", 38, false, "why?", "Who", "what", {frinds: "Hala"}];
+
+console.log('The length of the first array is ...' + firstArray.length);
+console.log('The length of the second array is ...' + secondArray.length);
+
+if(firstArray.length == secondArray.length){
+    console.log("They are the same!")
+}else{
+    console.log("Two different sizes")
```
Comment (marcianoviereck) [inline Week1/homework/js-exercises/exercises10.js]: Don't forget the `;` :)

File: Week1/homework/js-exercises/exercises8.js [hunk 4]
```diff
@@ -0,0 +1,69 @@
+'use strict';
+        const name = 'Amna'; //string
```
Comment (marcianoviereck) [inline Week1/homework/js-exercises/exercises8.js:2]: What happened here with the formatting:)?

File: Week1/homework/js-exercises/exercises8.js [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+            console.log('all of this are not different');
+        } else{
+                console.log('lastName is string but secondObject is an object');
+        }
+
+
+        console.log(typeof name);
+        console.log(typeof firstObject);
+
+        console.log(typeof name, typeof firstObject);
```
Comment (marcianoviereck) [inline Week1/homework/js-exercises/exercises8.js:69]: The rest of the assignment looks nice though:)

**Review by marcianoviereck — CHANGES_REQUESTED** [review]
The homework is looking nice, I do have some minor remarks though!

File: Week1/homework/js-exercises/exercise2.js (hunk 1 above)
Comment (AmnaAhmad) [reply Week1/homework/js-exercises/exercise2.js]: 'use strict';
console.log("I'm awesome");

File: Week1/homework/js-exercises/exercises8.js (hunk 5 above)
Comment (AmnaAhmad) [reply Week1/homework/js-exercises/exercises8.js:69]: thank you

**Review by AmnaAhmad — COMMENTED** [review]
hallo, have i anything else to make it again? 
thank you in advance.

**Pull request closed without merging**